content/en/docs/instrumentation/rust/_index.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Rust
-description: A language-specific implementation of OpenTelemetry in Rust.
+description: >-
+  <img width="35" src="https://github.com/open-telemetry/opentelemetry.io/blob/main/iconography/32x32/Rust.svg"></img>
```

Comment:
Update with URL to raw.github:

https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Rust.svg

Reply:
Done